tests/test_api.py

```diff
@@ -126,7 +126,7 @@ def dummy_api_function():
             utils.get_repo_info(TEST_GIT_URI, TEST_GIT_REF)
 
         # Check we get the correct exception
-        with pytest.raises(MethodCallError):
+        with pytest.raises(OsbsException):
```

Comment:
How did this go from MethodCallError to OsbsException? Are we just checking a less specific error type now?

Reply:
because older version failed just with MethodCallError, but new flexmock propagates error to osbsapi and raises OsbsException